astro/src/content/blog/why-passkeys-matter.mdx

```diff
@@ -0,0 +1,42 @@
+
+We need something better. Fortunately, [passkeys](/guides/what-is-a-passkey) are here as the answer. They're the replacement for passwords. As unlikely as the combination may seem, passkeys enhance security without sacrificing user experience.
+
+## The Limitations of Passwords
+
+If everyone followed best practices for passwords, we wouldn't need to have this conversation. Using only capital and lowercase letters, there are over 218 trillion possible password combinations. When you add numbers and symbols, that number increases exponentially. Seems secure enough, right?
+
+But people aren't very good at creating passwords. Upwards of 13 percent of Americans [use the same password](https://explodingtopics.com/blog/password-stats) for every account. Only 37 percent even bother using two-factor authentication. These behaviors make for insecure logins that are vulnerable from their creation.
+
+The [best identity and login solutions](/docs/customize/look-and-feel/client-side-password-rule-validation) make it easy for developers to require good password behaviors. They use rules, validated by the app itself. These rules force specific combinations of uppercase, lowercase, numbers, letters, and symbols. Unfortunately, the more secure password rules are, the more likely it becomes that people will recycle them, forget them, or generally find them unappealing.
+
+## The Rise of Passkeys
+
+Passkeys evolved out of the ideas behind public-key cryptography. This technology, first discussed in the 1970s, uses two keys. The public key is shared out in the open. The private key remains secure and confidential. The user can only gain access when both keys are present.
+
+Fast forward to 2022 and big names in technology (Microsoft, Google, and Apple) joined up to unveil their support for passkeys on World Password Day. Now the challenge is getting the rest of the world to follow suit.
+
+## Passkeys in Action
+
+If you've used Windows Hello or Apple's FaceID, you've probably used passkeys. That PIN, fingerprint, or facial scan proves your identity. Once that's done, you're then given access to the public and private key pair to unlock the app in question.
+
+Let's get specific. Say that you are signing up for access to a website. You used to input a username and password combination that you'd have to remember. With passkeys, that same website says "this person is logged in to a secure device, and we can trust that they are who they say they are." Instead of using a username and password combination, you scan your fingerprint, face, or use a PIN to gain access to the site moving forward.
```

Comment:
This is correct but seems a bit weird for me to read.  Just a suggestion.

```suggestion
Let's get specific. Say that you are signing up for access to a website. Typically, you would input a username and password combination that you'd have to remember. With passkeys, that same website says "this person is logged in to a secure device, and we can trust that they are who they say they are." Instead of using a username and password combination, you scan your fingerprint, face, or use a PIN to gain access to the site moving forward.
```